src/software/ai/hl/stp/play/defense/defense_play_base.h

```diff
@@ -45,13 +45,17 @@ class DefensePlayFSMBase
     void setUpPassDefenders(int num_pass_defenders);
 
     /**
-     * Helper function to set up alignments for the crease defense robots
+     * Helper function to set up alignments for the crease defense robots and control
+     * params
      *
      * @param event the FSM event
      * @param crease_defender_assignments crease defender assignments to be aligned
+     * @param ball_steal_mode crease defender ball steal behaviour/aggressiveness
      */
-    void setAlignment(const Update &event,
-                      const std::vector<DefenderAssignment> &crease_defender_assignments);
+    void setAlignment(
+        const Update &event,
+        const std::vector<DefenderAssignment> &crease_defender_assignments,
+        TbotsProto::BallStealMode ball_steal_mode = TbotsProto::BallStealMode::STEAL);
```

Comment:
I wonder if we should not have a default value so we can always make the steal mode explicit. What do you think?

Reply:
I considered this as well, and I think that is a good point for setAlignment. I think I will modify that for this function since it is a component of the defence play (it makes sense for a play to be clear in its defense strategy). But more importantly, I also have this default steal setting in place within the crease tactic control params. I think there are two strong reasons why a default value would be preferred for this tactic:
1) In general, we want our crease defenders to have this stealing behaviour. Aside from a few exceptions such as in stop play. 
2) Simplicity sake. This setting is not really integral to each crease that exists and really just helps us deal with the 1 or 2 edge cases that come up.